mappings/net/minecraft/block/entity/PistonBlockEntity.mapping

```diff
@@ -66,6 +66,6 @@ CLASS net/minecraft/unmapped/C_wpearmnv net/minecraft/block/entity/PistonBlockEn
 	METHOD m_szmbtwln finish ()V
 	METHOD m_tlxmitvb getRenderOffsetX (F)F
 		ARG 1 tickDelta
-	METHOD m_usuotmqt getMovementDirection ()Lnet/minecraft/unmapped/C_xpuuihxf;
+	METHOD m_usuotmqt getMoveDirection ()Lnet/minecraft/unmapped/C_xpuuihxf;
```

Comment:
I'd keep it as movement

Reply:
changed this and the other two instances (back) to 'movement'